l1-contracts/contracts/bridge/interfaces/IL1SharedBridge.sol

```diff
@@ -15,26 +16,27 @@ interface IL1SharedBridge {
         bytes32 indexed l2DepositTxHash,
         address indexed from,
         address to,
-        address l1Token,
+        address l1Asset,
         uint256 amount
     );
 
     event BridgehubDepositInitiated(
         uint256 indexed chainId,
         bytes32 indexed txDataHash,
         address indexed from,
-        address to,
-        address l1Token,
-        uint256 amount
+        bytes32 assetId,
+        bytes32 bridgeMintCalldataHash
     );
 
     event BridgehubDepositBaseTokenInitiated(
         uint256 indexed chainId,
         address indexed from,
-        address l1Token,
+        bytes32 assetId,
         uint256 amount
     );
 
+    event BridgehubMintData(bytes bridgeMintData);
```

Comment:
this event is never emitted. Please go over all other events and ensure that these are used